benchmark/storage/sql_table_benchmark.cpp

```diff
@@ -107,9 +124,131 @@ BENCHMARK_DEFINE_F(SqlTableBenchmark, SimpleInsert)(benchmark::State &state) {
   state.SetItemsProcessed(state.iterations() * num_inserts_);
 }
 
-// Insert the num_inserts_ of tuples into a DataTable in a single thread
+// Insert the num_inserts_ of tuples into a SqlTable concurrently
+// NOLINTNEXTLINE
+BENCHMARK_DEFINE_F(SqlTableBenchmark, ConcurrentInsert)(benchmark::State &state) {
```

Comment:
I am trying understand how these benchmarks work. What does State represent? I couldn't find it defined anywhere

Reply:
It's a google benchmark (gbenchmark). 
https://github.com/google/benchmark